pkg/apis/internal.open-cluster-management.io/v1beta1/clusterinfo_types.go

```diff
@@ -29,22 +29,26 @@ type ClusterInfoStatus struct {
 	Version string `json:"version,omitempty"`
 
 	// KubeVendor describes the kubernetes provider of the managed cluster.
+	// Deprecated in release 2.3. Use clusterClaim platform.open-cluster-management.io instead.
```

Comment:
add  a comment that this is to be removed in the future

Reply:
done